### DISCUSSION TITLE
Verification case WUI/hot_rods.fds

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#13907] [Q&A]
Dear team,
In the Verification case `WUI/hot_rods.fds`, you said that the radius of rods were uniformly distributed in the range of 0.002 and 0.003 m. Meanwhile, it also refered that $f(r)=1000$, $\bar{r^2} = \int_{0.002}^{0.003}r^2 f(r) dr \approx 6.333\times 10^{-6}$. I'm confused about the meaning of $f(r)$ and how to get the distributed ranges.
LI

**Comment by mcgratta** [comment]
The expression $\overline{r^2}$ is known as the _expected_ value of $r^2$. The expected value of the radius, $r$, denoted by $\overline{r}$ or $E(r)$, is 0.0025, but the expected value of $r^2$ is a little more than $\overline{r}^2$.  Another way to think about it is to say that the mean area of the circles is not $\pi$ times the mean radius squared.

**Reply by lihy213** [reply]
My doubt is that there's no information about the range of radius in the input file `hot_rod.fds`, so how to determine its range between 0.002 and 0.003 m? Besides, I still don't know the meaning of $f(r)$.

**Comment by drjfloyd** [comment]
The CNF_RAMP on PART points to a RAMP that defines the number distrubution as a function of the diamter in $\mu m$. 

This RAMP says the CNF is uniform between diameters of 4000 and 6000 $\mu m$ or a radius of 0.002 to 0.003 m

&RAMP ID='dist', T=4000., F=0. /
&RAMP ID='dist', T=6000., F=1. /

**Reply by lihy213** [reply]
Ohhh! I got it!
